frontend_app_html-document/src/css/index.styl

```diff
@@ -24,7 +24,7 @@
       margin 0 pageSpacing
     &__left
       &__wrapper
-        height "calc(100% - %s)" % (headerHeight + standardSpacing)
+        height "calc(100% - %s)" % (headerHeight + standardSpacing + 28px) // 28px is the option menu height
```

Comment:
Often, height calc(100% - ...) can be replaced by flex 1 (maybe not applicable here but that's something I noticed in Tracim's frontend code multiple time, so probably good to know for the future).

Reply:
thx ! fixed.